src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -853,25 +849,22 @@ query_path(const char *szInput, int *is_target_path, dfs_obj_t **parent, char *i
 		pt_end = stpncpy(full_path_parse, cur_dir, DFS_MAX_PATH);
 		len = (int)(pt_end - full_path_parse);
 		if (len >= DFS_MAX_PATH) {
-			if (daos_inited && bLog)
-				D_ERROR("full_path_parse[] is not large enough.");
+			D_ERROR("full_path_parse[] is not large enough.\n");
```

Comment:
Again, think about how a user should respond if they encounter this message, it reads like a debug statement, not an error message.

Reply:
Fixed as suggested. Thank you!